### ISSUE TITLE
Add zoom link to recurring calendar invite

### ISSUE THREAD

**Issue by bmuenzenmeyer** [nodejs/build#3957]
> [!NOTE]  
> This is cleanup from https://github.com/nodejs/admin/issues/923

People interacting with the WG via https://nodejs.org/calendar are met with not enough information to join.

![image](https://github.com/user-attachments/assets/87c4d26a-1430-4d7a-bb7e-393f4e367dec)

And they won't necessarily know to look for issues like https://github.com/nodejs/build/issues/3942

Further https://nodejs.org/en/about/get-involved/events only displays events with a zoom link.

To resolve all of this, please add https://zoom.us/j/715960833 (which I think is a durable enough link, taken from the above issue) to the calendar.

**Comment by richardlau** [comment]
@bmuenzenmeyer most Working Groups won't be able to edit their calendar entries. I don't know if the list is still accurate after the rebuild in https://github.com/nodejs/admin/issues/923 but https://github.com/nodejs/admin?tab=readme-ov-file#calendar-maintainers lists the people who are supposed to be able to edit the calendar.

**Comment by bmuenzenmeyer** [comment]
I agree, but these issues are for, first awareness of the problem and impact,  and then action. 

I mentioned the same list in Slack, but didn't want to outright ping everyone.

**Comment by mhdawson** [comment]
Added the zoom link to the calendar.
